lib/vmpooler/pool_manager.rb

```diff
@@ -437,16 +445,25 @@ def _clone_vm(pool_name, provider, request_id = nil, pool_alias = nil)
           start = Time.now
           provider.create_vm(pool_name, new_vmname)
           finish = format('%<time>.2f', time: Time.now - start)
+          $logger.log('s', "[+] [#{pool_name}] '#{new_vmname}' cloned in #{finish} seconds")
+          $metrics.timing("clone.#{pool_name}", finish)
+
+          $logger.log('d', "[ ] [#{pool_name}] Obtaining IP for '#{new_vmname}'")
+          ip_start = Time.now
+          ip = provider.get_vm_ip_address(new_vmname, pool_name)
+          ip_finish = format('%<time>.2f', time: Time.now - ip_start)
+          $logger.log('s', "[+] [#{pool_name}] Obtained IP for '#{new_vmname}' in #{ip_finish} seconds")
```

Comment:
Ya since pool manager doesn't currently implement any tracing, probably better as a separate task. I opened https://puppet.atlassian.net/browse/POOLER-215 for that.